python/artm/batches_utils.py

```diff
@@ -276,9 +276,13 @@ def __reset_batch():
     @property
     def batches_list(self):
         """
-        :return: list of batches names
```

Comment:
@MelLain Unfortunately, right now changing logic of `batches_list` property breaks code which uses it rather weird way, e.g. `hARTM` :( In my opinion, right now it's ok to use simple hack with `process_in_memory` until we understand problem with `hARTM` class